code/modules/clothing/under/miscellaneous.dm

```diff
@@ -765,6 +765,7 @@
 	item_state = "atmos_suit"
 	item_color = "atmos"
 	burn_state = FIRE_PROOF
+	antag_hints = "<span class='traitorhint'>This jumpsuit would let you contort into vents and other tight spaces.</span>"

```

Comment:
This jumpsuit allows the wearer to contort themselves into vents and other tight spaces.